### PR TITLE
[Red Glitchless] Add "intermediate route"

### PR DESCRIPTION
**Pull request by SquareSheepofRoot**
Adding intermediate red route. Includes a pastebin linked to simplified backup boom strats - ideally this could become a document under resources in the future.

### CODE REVIEW

**Comment by SquareSheepofRoot** [comment]
Just made an edit to weedle guy fight & several formatting edits.

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+- HA x3-4(finish with Tackle if you can)
+
++ Catch a Flyer
+  + Tackle then toss a ball at any flyer except L8 Pidgey where you HA then toss a ball
+  > Exception: if only 2 HAs are left, then weaken L8 Pidgey with Leer + Tackle
+
+## MT. MOON SPLIT
+
+**Moon Manip Save**
+- 1- 8
```
Comment (hwangbro) [inline docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Thoughts on simplifying this to "heal to 36+"

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md [hunk 2]
```diff
@@ -0,0 +1,714 @@
...
+- 9 - 15
+  + 2 Potions
+- 16 - 35
+  + 1 Potion
+- 36+
+  + Don't Heal
+
+Do Route 3 Moon Manip https://pastebin.com/tggXpQRC
+
+>The Lass that you see on screen before entering Mt. Moon during the manip should be stuck facing up; however on Frame 36 & Frame 37 she walks right & faces left, respectively. If you'd like to learn the alternative movement used to manipulate mt. moon on these frames check here (../../../resources/route-3-moon-backups.md)
```
Comment (hwangbro) [inline docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Can you make the f36/37 reference a hyperlink?

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md [hunk 3]
```diff
@@ -0,0 +1,714 @@
...
+  + @-2 hp < 14; if confused yolo or consider swap on good pace if Bird or Squirtle have low hp
+
+**Moon Menu** immediately after defeating Rocket:  
+  + Toss Pokeball(s) (do not toss if you still need to catch Paras)
+  + Use Rare Candy
+  + Teach Water Gun (TM12) in Slot 2
+  + Use Moon Stone
+  + Teach Mega Punch (TM01) in Slot 1
+
+> Moon Exp: KO a Geodude or Paras if you happen to encounter one while walking up to talk to the Nerd
```
Comment (hwangbro) [inline docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Maybe superfluous, mention that you use PS to kill Paras?

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md [hunk 4]
```diff
@@ -0,0 +1,714 @@
...
+
+## LORELEI SPLIT
+
+Super Repel + Bike
+
+Rival:
+- X Acc + X Speed + Blizzard + (TB)
+- HD x5
+
+Pick up the hidden Max Ether
```
Comment (hwangbro) [inline docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Should link to pokeworld here

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md [hunk 5]
```diff
@@ -0,0 +1,714 @@
...
++ **(27-48)** Super Potion x2 
++ **(49-56)** Super Potion & Potion x2
++ **(57-76)** Super Potion & Potion
++ **(77-98)** Super Potion
++ **(99-106)** Potion x2
++ **(107-125)** Potion
++ **(126+)** Don't Heal
+
+Lance:
+- X Special + TB (if hydro pump misses you can stall with EQ or with an X-Acc if you still have 2 left )
```
Comment (hwangbro) [inline docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Maybe mention x speed stall hps here as well

**Review by hwangbro — CHANGES_REQUESTED** [review]

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md (hunk 1 above)
Comment (SquareSheepofRoot) [reply docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Yes, did this and added an if confident with yolo ball under 9 hp then delay heal clause.

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md (hunk 2 above)
Comment (SquareSheepofRoot) [reply docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Yes fixed this link.

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md (hunk 3 above)
Comment (SquareSheepofRoot) [reply docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Yup, makes sense. Added WG geodude as well cause why not

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md (hunk 4 above)
Comment (SquareSheepofRoot) [reply docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Yes added it.

File: docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md (hunk 5 above)
Comment (SquareSheepofRoot) [reply docs/gen-1/red-blue/main-glitchless/intermediate-route/README.md]: Yes, HP < 133 right?

**Review by hwangbro — APPROVED** [review]

**Pull request merged**